common/cosmwasm-smart-contracts/mixnet-contract/src/reward_params.rs

```diff
@@ -105,7 +105,7 @@ pub struct EpochRewardParams {
     epoch_reward_pool: Uint128,
     rewarded_set_size: Uint128,
     active_set_size: Uint128,
-    circulating_supply: Uint128,
+    staking_supply: Uint128,
```

Comment:
You want to put some `serde(alias)` action here so that you would be able to deserialize old data from the `circulating_supply` field

Reply:
wow, thats a great idea!